### PR TITLE
Feature/verify token

### PR DESCRIPTION
**Pull request by tleekkul**
Add verify token API that user gets from Odin's 3rd party login page.

### CODE REVIEW

File: Configuration/odin-app.json [hunk 1]
```diff
@@ -15,6 +15,9 @@
                         "expires": {"hours": 72},
                         "static": "usr/share/odin/Static/app/login.html"
                     }
+                },
+                "verify/": {
```
Comment (KayEss) [inline Configuration/odin-app.json]: This name looks wrong. Whouldn't there be `views/` at the front of it?

File: Cpp/odin-views/app.cpp [hunk 2]
```diff
@@ -93,6 +95,91 @@ namespace {
...
+            if ( paths.size() != 1)
+                throw fostlib::exceptions::not_implemented(__PRETTY_FUNCTION__,
+                    "Must pass app_id in the URL");
+            fostlib::pg::connection cnx{fostgres::connection(config, req)};
+            const auto app_id = paths[0];
+            fostlib::json app = odin::app::get_detail(cnx, app_id);
+            if ( app.isnull() )
+                throw fostlib::exceptions::not_implemented(__PRETTY_FUNCTION__, "App not found");
+
+            if ( req.method() != "POST" )
```
Comment (KayEss) [inline Cpp/odin-views/app.cpp]: Really we should have braces, `{}`, around the body of the `if`. There's a few places here where they're missing.

File: Cpp/odin/app.cpp [hunk 3]
```diff
@@ -53,5 +56,17 @@ fostlib::jwt::mint odin::app::mint_user_jwt(
...
     return jwt;
 }
+
+void odin::app::save_app_user(
+    fostlib::pg::connection &cnx, f5::u8view reference,
+    const fostlib::json &user, const f5::u8view app_id
+) {
+    fg::json app_user_values;
+    fostlib::insert(app_user_values, "reference", reference);
+    fostlib::insert(app_user_values, "identity_id", user["identity"]["id"]);
```
Comment (KayEss) [inline Cpp/odin/app.cpp]: Probably better to have the `identity_id` passed in like we do for the `app_id`, then we're not reliant on particular JSON structures.

File: Configuration/odin-app.json (hunk 1 above)
Comment (tleekkul) [reply Configuration/odin-app.json]: It's a part of `pathprefix` view configuration, not views

**Pull request closed without merging**